lpr.Common/Models/Participant.cs

```diff
@@ -11,7 +11,6 @@ public class Participant
         public Guid Id {  get; set; }
         public Organisation? Organisation { get; set; }
         public Package? Package {  get; set; }
-        public List<Role> Roles { get; set; }
```

Comment:
I understand certain properties aren't yet implemented, but they will be in the future.

Reply:
Yes, something will be implemented. But at the moment there is no decision made if we are going to create our own Right-Role system or going to use the Right-Roles system from ASP.NET.